### PR TITLE
Add jest tests for twenty-front

### PR DESCRIPTION
**Pull request by gitstart-twenty**

### CODE REVIEW

File: packages/twenty-front/src/hooks/__tests__/useCombinedRefs.test.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { renderHook } from '@testing-library/react';
+
+import { useCombinedRefs } from '~/hooks/useCombinedRefs';
+
+describe('useCombinedRefs', () => {
```
Comment (gitstart-twenty) [inline packages/twenty-front/src/hooks/__tests__/useCombinedRefs.test.ts:6]: @charlesBochet There's lots of "complex" hooks. I suppose we're writing tests for them too. I suggest doing it in smaller PRs to make reviews much easier.

**Review by gitstart-twenty — COMMENTED** [review]
Overall, what do you think of the concept of smaller PRs, sub(child) issues from the upcoming "big" ticket?

File: packages/twenty-front/src/hooks/__tests__/useCombinedRefs.test.ts (hunk 1 above)
Comment (charlesBochet) [reply packages/twenty-front/src/hooks/__tests__/useCombinedRefs.test.ts:6]: Let's track the whole work within #2992. We can discuss on this ticket what next hooks should be tackled

**Review by charlesBochet — APPROVED** [review]
I have fixed an additional test, merge main into your branch and added a case to getOnboardingStatus test as we modified it in main recently

**Comment by charlesBochet** [comment]
Thanks!

**Pull request merged**